CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 
 ### Added
 
+- Internal: Now the UI tests will hit API endpoints from a dockerised mock server.
```

Comment:
Looks like this `CHANGELOG` note has ended up merged into `6.4.0` release notes.

Reply:
I think we're still good: https://github.com/onfido/onfido-sdk-ui/blob/master/CHANGELOG.md#added

Reply:
Yeah, just something to look out for when it's merged into `development`